doc/ref_cert/RC2/chapters/chapter02.md

```diff
@@ -255,6 +262,9 @@ skip:
   - [Feature:Volumes]
   - [Feature:Windows]
   - [NodeFeature:EphemeralStorage]
+  - PersistentVolumes.NFS
+  - should provision storage with different parameters
```

Comment:
Do we need to include the exclusions introduced above again under the "skip" section here?

Reply:
Same question here. This looks confusing to me.

Reply:
@rihabbanday @gkunz I could remove them if you prefer. My point was to list the skip list as written in testcases.yml and to precise any regex which are not clear from RA2 sig tabs.